whylog/tests/tests_config/test_files/parsers.yaml

```diff
@@ -5,17 +5,20 @@ name: lostdata
 primary_key_groups: [1]
 regex_str: '^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Data is missing at (.*)\. Loss = (.*)
   GB\. Host name: (.*)$'
+line_content: "2016-04-12 23:54:43 Data is missing at comp2. Loss = 150 GB. Host name: host2"
```

Comment:
line_content is to give you example of line that matches regex.

Suppose you want to build a new rule and want to use parser that was added one day to Whylog knowledge base. You can search it by its name. Ok, you found it. Suppose that there is no line_content... Then you see only regex. Regex can be very strange, difficult to understand sometimes. And that is why we need line_content. It is here is to give you example of line that matches this strange regex.